apis/coursebatchmanapi/coursebatchmanapi.yaml

```diff
@@ -56,7 +56,843 @@ paths:
         required: true
         type: string
         description: >-
-            Type of the request data. Set as application/json.
+            The Content Type entity is the media type of the resource.Possible media types can be:- 

```

Comment:
The Content-Type entity represents the media type of the resource. 
Possible media types can be- need more inputs on this